README.md

```diff
@@ -11,7 +11,9 @@
 
   [![npm](https://img.shields.io/npm/v/langium)](https://www.npmjs.com/package/langium)
   [![Build](https://github.com/langium/langium/actions/workflows/build.yml/badge.svg)](https://github.com/langium/langium/actions/workflows/build.yml)
-  [![Gitpod Ready-to-Code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/langium/langium)
+  [![Gitter Chat](https://img.shields.io/badge/chat-on%20gitter-0DBD8B?logo=gitter)](https://app.gitter.im/#/room/#langium:gitter.im)
+  [![Github Discussions](https://img.shields.io/badge/github-discussions-blue?logo=github)](https://app.gitter.im/#/room/#langium:gitter.im)
```

Comment:
Seems this link still goes to the gitter chat.

Reply:
Yep, thanks 👍